corehq/apps/userreports/sql.py

```diff
@@ -101,3 +104,33 @@ def _hash(domain, table_id):
 
     return 'config_report_{0}_{1}_{2}'.format(domain, table_id, _hash(domain, table_id))
 
+
+def get_expanded_columns(table_name, column_config):
+
+    session = Session()
+    connection = session.connection()
+    metadata = sqlalchemy.MetaData()
+    metadata.reflect(bind=connection)
+
+    column = metadata.tables[table_name].c[column_config.field]
+    query = sqlalchemy.select([column]).distinct()
+
+    result = connection.execute(query).fetchall()
```

Comment:
@snopoke should the connection be closed as well, or does closing the session also close the connection?